src/main/resources/reference.conf

```diff
@@ -152,9 +152,13 @@ akka {
 # * visible: Boolean (false by default)
 # * description: String
 #
-# Additionally, on attributes level you can provide two parameters:
+# Additionally, on attributes level you can provide following parameters:
 # * placeholder: String
 # * data-format: String
+# * value-map: Map[String, String], example: {"0": "Yay", "1": "Nay", "2": "Pass"}
+# * data-type: String, since this value is later mapped to enum, values are limited to those supported by method
+#                      tech.cryptonomic.conseil.tezos.TezosPlatformDiscoveryOperations.mapType
+# * scale: Int
```

Comment:
Maybe this could deserve some explanation or, even best, an example?

Reply:
done